.gitignore

```diff
@@ -20,6 +20,7 @@ man/*.1
 
 # Distribution / packaging
 .Python
+.idea/
```

Comment:
.idea is for PyCharm and ,cover is for some coverage data.
If you create a new Python project on GitHub, those are already in your `.gitignore`.